source/Coop.Core/Client/Services/MapEvent/Handlers/StartBattleClientHandler.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Common.Logging;
+using Common.Messaging;
+using Common.Network;
+using Coop.Core.Client.Services.Clans.Handler;
+using Coop.Core.Client.Services.MapEvent.Messages;
+using Coop.Core.Server.Services.Clans.Messages;
+using Coop.Core.Server.Services.MapEvent.Messages;
+using GameInterface.Services.Clans.Messages;
+using GameInterface.Services.MapEvents.Messages;
+using Serilog;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Coop.Core.Client.Services.MapEvent.Handlers
+{
+    /// <summary>
+    /// Client handler for Starting Map Events
+    /// </summary>
+    public class StartBattleClientHandler : IHandler
```

Comment:
Could you go through all the client handler names an update to the following
```suggestion
    public class ClientStartBattleHandler : IHandler
```